content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -86,7 +86,7 @@ spec:
                code:
                  remote:
                    http_uri:
-                     uri: https://github.com/istio-ecosystem/wasm-extensions/releases/download/{{< istio_version >}}.0/basic-auth.wasm
+                     uri: https://github.com/istio-ecosystem/wasm-extensions/releases/download/{{< istio_version >}}.1/basic-auth.wasm
```

Comment:
I'm not sure if there is a better mechanism to get the latest version updated automatically.

Reply:
I think that extension will likely remain static for a very long time in the future, so the need to bump its version is very low. Automating this probably won't be very valuable.

Reply:
Also not sure of a good way to handle this across release boundaries. For example, preliminary.istio.io has: `https://github.com/istio-ecosystem/wasm-extensions/releases/download/1.10.0/basic-auth.wasm` which isn't a valid release at the moment. Does it make sense to remove istio_version?

Reply:
The basic auth extension is tested with the corresponding Istio release, so I think it would be safe to keep the doc followig Istio release.

I am going to work on prow automation to release Wasm extension from the `istio-ecosystem/wasm-extensions` repo, so that when Istio has a release branch cut, a new release will be made from that repo, which makes sure that the istio.io doc will always be valid at the release time.

Reply:
> The basic auth extension is tested with the corresponding Istio release, so I think it would be safe to keep the doc followig Istio release.
> 
> I am going to work on prow automation to release Wasm extension from the `istio-ecosystem/wasm-extensions` repo, so that when Istio has a release branch cut, a new release will be made from that repo, which makes sure that the istio.io doc will always be valid at the release time.

So preliminary.istio.io will be valid for the time from branch cut to release (we cut the new release-1.10 branch you create the new extensions), but from release to the next branch cut preliminary.istio.io will not have a valid link (as it will expect a 1.11.0 (similar to expecting a 1.10.0 now).

Reply:
> So preliminary.istio.io will be valid for the time from branch cut to release (we cut the new release-1.10 branch you create the new extensions), but from release to the next branch cut preliminary.istio.io will not have a valid link (as it will expect a 1.11.0 (similar to expecting a 1.10.0 now).

Yeah.. would this be problemtic?